--- origsrc/mysql-connector-odbc-5.1.8/driver/Makefile.am	2010-10-28 13:33:18.000000000 -0500
+++ src/mysql-connector-odbc-5.1.8/driver/Makefile.am	2011-01-23 21:24:24.445004500 -0600
@@ -24,7 +24,7 @@ AUTOMAKE_OPTIONS=foreign
 
 lib_LTLIBRARIES=libmyodbc5.la
 
-INCLUDES=-I$(top_srcdir)/util -I$(top_srcdir)
+INCLUDES=-I$(top_builddir) -I$(top_srcdir)/util -I$(top_srcdir)
 
 ###################################################################
 #                                                                 #
--- origsrc/mysql-connector-odbc-5.1.8/installer/Makefile.am	2010-10-28 13:33:18.000000000 -0500
+++ src/mysql-connector-odbc-5.1.8/installer/Makefile.am	2011-01-23 21:00:42.066926200 -0600
@@ -18,7 +18,7 @@
 
 if MYODBCINST
 
-INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/util
+INCLUDES = -I$(top_builddir) -I$(top_srcdir) -I$(top_srcdir)/util
 
 bin_PROGRAMS = myodbc-installer
 myodbc_installer_SOURCES = myodbc3i.c
--- origsrc/mysql-connector-odbc-5.1.8/util/Makefile.am	2010-10-28 13:33:17.000000000 -0500
+++ src/mysql-connector-odbc-5.1.8/util/Makefile.am	2011-01-23 20:39:48.687137300 -0600
@@ -1,4 +1,4 @@
-INCLUDES = -I$(top_srcdir)
+INCLUDES = -I$(top_builddir) -I$(top_srcdir)
 
 noinst_LTLIBRARIES = libmyodbc3u.la
 
