--- origsrc/mysql-connector-odbc-5.1.8/driver/Makefile.am	2010-10-28 13:33:18.000000000 -0500
+++ src/mysql-connector-odbc-5.1.8/driver/Makefile.am	2011-01-23 21:24:24.445004500 -0600
@@ -61,7 +61,7 @@ driver.h
 ###################################################################
 libmyodbc5_la_CPPFLAGS       = -DNONTHREADSAFE
 
-libmyodbc5_la_LDFLAGS       = -release @NUMERIC_VERSION@ -module 
+libmyodbc5_la_LDFLAGS       = -module -avoid-version -no-undefined
 
 if WITH_DEBUG
 LTLIBS_DEPS = $(LTLIBS_DEBUG_DEPS)
